openshift-knative-operator/pkg/serving/extension_test.go

```diff
@@ -187,6 +187,26 @@ func TestReconcile(t *testing.T) {
 				},
 			}
 		}),
+	}, {
+		name: "respect kourier settings",
+		in: &v1alpha1.KnativeServing{
+			Spec: v1alpha1.KnativeServingSpec{
+				Ingress: &v1alpha1.IngressConfigs{
+					Kourier: v1alpha1.KourierIngressConfiguration{
+						// Enabled: true omitted explicitly.
+						ServiceType: corev1.ServiceTypeClusterIP,
```

Comment:
Do we cover the case where all ingresses are disabled?

Reply:
This is that case, technically. The `Enabled` field is false on all Ingresses.

Reply:
Yeah it is implicit due to default false value. How about the nil case?

Reply:
Tested as the default case in pretty much every other test.